sql/core/benchmarks/JsonBenchmark-results.txt

```diff
@@ -3,127 +3,128 @@ Benchmark for performance of JSON parsing
 ================================================================================================
 
 Preparing data for benchmarking ...
-OpenJDK 64-Bit Server VM 17.0.8+7-LTS on Linux 5.15.0-1046-azure
+OpenJDK 64-Bit Server VM 17.0.8+7-LTS on Linux 5.15.0-1047-azure
```

Comment:
Sorry for the late reply, but I have to say, we also need to update `JsonBenchmark-jdk21-results.txt` ... @panbingkun

Reply:
Okay,I will sumbit a followup pr to complete it

Reply:
Followup pr: https://github.com/apache/spark/pull/43346